mip/var.go

```diff
@@ -0,0 +1,61 @@
+package mip
+
+// Var represents the entities on which the solver has to make a decision
+// without violating constraints and optimize the objective.
```

Comment:
```suggestion
// without violating constraints and while optimizing the objective.
```
Always take my English advice with a grain of salt. :joy_cat: